$NetBSD: patch-Makefile,v 1.5 2024/11/29 22:55:50 wiz Exp $

install(1) on NetBSD does not support (this) -Dt nor --mode.
Sent upstream.

--- Makefile.orig	2024-10-25 23:17:54.000000000 +0000
+++ Makefile
@@ -1004,7 +1004,7 @@ install-pkgxdg: src/tuxpaint.desktop src
 	cp -a data/images/icon96x96.png $(NEWICON_PREFIX)/96x96/apps/tuxpaint.png
 	cp -a data/images/icon128x128.png $(NEWICON_PREFIX)/128x128/apps/tuxpaint.png
 	cp -a data/images/icon192x192.png $(NEWICON_PREFIX)/192x192/apps/tuxpaint.png
-	install --mode=0644 -Dt $(METAINFO_PREFIX) src/org.tuxpaint.Tuxpaint.appdata.xml
+	cp -a src/org.tuxpaint.Tuxpaint.appdata.xml $(METAINFO_PREFIX)
 
 .PHONY: install-xdg
 install-xdg: src/tuxpaint.desktop src/tuxpaint-fullscreen.desktop src/org.tuxpaint.Tuxpaint.appdata.xml
@@ -1039,7 +1039,8 @@ install-xdg: src/tuxpaint.desktop src/tu
 	@if [ "x$(shell which update-desktop-database)" != "x" ]; then \
 	  update-desktop-database ; \
 	fi
-	install --mode=0644 -Dt $(METAINFO_PREFIX) src/org.tuxpaint.Tuxpaint.appdata.xml
+	mkdir -p $(METAINFO_PREFIX)
+	cp -a src/org.tuxpaint.Tuxpaint.appdata.xml $(METAINFO_PREFIX)
 #do package path related stuff, ie use destdir
 
 # Install the PNG icon (for KDE desktop, etc.)
